x-pack/docs/en/security/operator-privileges/configure-operator-privileges.asciidoc

```diff
@@ -0,0 +1,61 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[configure-operator-privileges]]
+=== Configure operator privileges
+
+==== Enable operator privileges
+Operator privileges need to be enabled explicitly on each node in the cluster.
+This is done by configure the `elasticsearch.yml` file:
+
+[source,yaml]
+----------------------------
+xpack.security.operator_privileges.enabled: true
+----------------------------
+
+If the node is already running before the above change is made, you need restart
+it for the feature to take effect.
+
+WARNING: The feature needs to be either enabled or disabled consistently across all nodes
+in a cluster. Otherwise, you can get inconsistent behaviours depending on which node
+a request hits first and where it gets executed.
+
```

Comment:
I think there's a section missing here. You'll probably want to clean up the words a bit, but roughly:

> When operator privileges are enabled on a cluster, [specific functionality](https://elasticsearch_68964.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/operator-only-functionality.html) is restricted so that it may only be executed by those used who have been explicitly designated as operators.

That just makes the flow of thought a little more clear -  it walks the reader through the process of
- turn on the setting,
- what did that do?
- okay now how do I how do I designate which users are intended to be operators?

Reply:
Added two sentences to make the transition smoother as suggested.